scripts/README.md

```diff
@@ -218,8 +218,7 @@ WAIT_FOR_KNATIVE=1
 
 initialize $@
 
-# TODO: use go_test_e2e to run the tests.
```

Comment:
Please revert, this TODO is intentional as this is a sample script.

Reply:
Done